src/hooks/useSwitchNetwork.ts

```diff
@@ -0,0 +1,37 @@
+import { useMutation } from "react-query";
+import { NetworkId } from "src/constants";
+import { networkIdToHexString } from "src/helpers";
+
+import { useAddNetwork } from "./useAddNetwork";
+import { useWeb3Context } from "./useWeb3Context";
+
+/**
+ * Switches the wallets currently active network.
+ */
+export const useSwitchNetwork = () => {
```

Comment:
can this replace `export const switchNetwork` in `src/helpers/NetworkHelper`

Reply:
Yep absolutely, once this hook is actually being used in the app i'll then remove `switchNetwork` from `src/helpers/NetworkHelper`.